src/box/lua/index.c

```diff
@@ -47,8 +47,11 @@ static uint32_t CTID_STRUCT_ITERATOR_PTR = 0;
 static int
 lbox_insert(lua_State *L)
 {
-	if (lua_gettop(L) != 2 || !lua_isnumber(L, 1))
-		return luaL_error(L, "Usage space:insert(tuple)");
+	if (lua_gettop(L) != 2 || !lua_isnumber(L, 1)) {
+		diag_set(ClientError, ER_ILLEGAL_PARAMS,
+			 "Usage space:insert(tuple)");
```

Comment:
Since the message is printed after a comma, let's start with a lower-case letter everywhere? I'd also try to unify the messages while we are at it: in some places we print "usage" while in others "use"; sometimes we put a colon after "usage" sometimes we don't.